fares-data-build-tool/create-data
# Feature/cfd 295 global settings page design

# Description

This ticket creates the user interface for the global settings page and also displays a link to "operator settings" on the home page provided it's not hosted on prod 

# Testing instructions

Spin the site up and head to http://localhost:5555/globalSettings and check it looks like the ticket says it should

# Type of change


# Checklist:
